lib/certs/cacert.pem

```diff
@@ -3212,3 +3212,61 @@ gLWi5h+xEk8blTAKBggqhkjOPQQDAwNoADBlAjEA31SQ7Zvvi5QCkxeCmb6zniz2C5GMn0oUsfZk
 vLtoURMMA/cVi4RguYv/Uo7njLwcAjA8+RHUjE7AwWHCFUyqqx0LMV87HOIAl0Qx5v5zli/altP+
 CAezNIm8BZ/3Hobui3A=
 -----END CERTIFICATE-----
+
+DigiCert Global Root G2
```

Comment:
this already exists in the file (ln 1759)

Reply:
I guess that is a different certificate. I don't know the usecase of the old one. But this was Provided by Authorize.net on their documentation page. Link is attached in the PR comment.

I downloaded this certificate from the official DigiCert website. Following is the link to the respective page.
https://www.digicert.com/kb/digicert-root-certificates.htm#:~:text=expired%20%C2%A0%C2%A0revoked-,DigiCert%20Global%20Root%20G2,-Download%20PEM%20%7C

Reply:
@dawoodmalhi, please remove this repeated certificate so bradbroge can reconsider merging this PR with the DigiCert certificate only.

Reply:
this certificate has been removed from the PR.